benchmarks/nemo/nemolite2d/manual_versions/psykal_kokkos/README.md

```diff
@@ -0,0 +1,63 @@
+# Manual Kokkos version of PSyKAl NEMOLite2D #
+
+This directory contains a version of the PSyKAl form of NEMOLite2D
+that has had its PSy layer (in `time_step_kokkos.cpp`) manually
+parallelised using Kokkos. The Fortran(NemoLite2d) to C++(PSy-layer)
+is done with the interface introduced in the `psykal_cpp` version.
+
+Kokkos is a Performance Portability EcoSystem for C++,
+a good introduction to its concepts and APIs can be found in their
+(wiki)[https://github.com/kokkos/kokkos/wiki].
+
+
+## Implementation limitations ##
+
+Kokkos has several abstraction to control where (execution spaces),
```

Comment:
s/abstraction/abstractions/ and s/,/and/

Reply:
Done